tests/dags/test_ireland_timeline_enrich_dag.py

```diff
@@ -0,0 +1,22 @@
+
+from sem_covid.entrypoints.ml_dags.ireland_timeline_enrich_dag import DAG_NAME
+
+
+def test_eu_ireland_enrich_dag_has_two_tasks_and_order(airflow_dag_bag):
+    dag = airflow_dag_bag.get_dag(dag_id=DAG_NAME)
+    assert dag is not None
+    tasks = dag.tasks
+    task_ids = list(map(lambda task: task.task_id, tasks))
+    assert {'dataset_preparation', 'dataset_enrichment'}.issubset(set(task_ids))
+
+    dataset_preparation = dag.get_task('dataset_preparation')
```

Comment:
I start to notice a lot of repeating code for asserting the dag structure. 
I propose to make some helper functions that do this assertions for us.